samples/msal-browser-samples/vue3-sample-app/README.md

```diff
@@ -48,6 +48,7 @@ This sample demonstrates how you can integrate `@azure/msal-browser` into your V
 You'll find the MSAL configuration and `PublicClientApplication` instantiation in `authConfig.ts`. It's very important that `PublicClientApplication` is initialized only once per pageload and as such should not be initialized inside any Vue components, but rather outside the context of Vue and passed in.
 
 The `clientId` is the most important, and only required parameter, as it maps to your app registration in the Azure Portal.
+The `authority` represents the Azure AD instance and tenant that MSAL.js will use the sign users in. This parameter controls the audience of your app.
```

Comment:
Don't we support B2C for Vue? Or is this kept simple because this is just a sample?

Reply:
Good point. It can sign users in with B2C, but we don't handle user-flows in this one. Let me clarify this.

Reply:
done